account-kit/infra/src/client/smartAccountClient.ts

```diff
@@ -169,6 +170,7 @@ export function createAlchemySmartAccountClient(
           transport: config.transport,
           gasEstimatorOverride: config.gasEstimator,
           feeEstimatorOverride: config.feeEstimator,
+          webhookData: config.webhookData,
```

Comment:
do you know if it makes sense to include a global `webhookData` field on the client? my assumption was that it would be set per-request. (i could be mistaken though.)

Reply:
reverting changes to be per-request